nokia/__init__.py

```diff
@@ -221,6 +221,10 @@ def get_sleep(self, **kwargs):
         r = self.request('sleep', 'get', params=kwargs, version='v2')
         return NokiaSleep(r)
 
+    def get_sleep_summary(self, **kwargs):
```

Comment:
Should startdateymd and enddateymd be required parameters for this method? They are required for the api.
http://developer.withings.com/oauth2/#tag/sleep%2Fpaths%2Fhttps%3A~1~1wbsapi.withings.net~1v2~1sleep%3Faction%3Dgetsummary%2Fget

Reply:
I would prefer that, but I kept the code in the style of the other calls.
I'm willing to do that work.

Reply:
That's a good point. Requiring the parameters in other methods would break backwards compatibility. How about raising an exception when those params are not provided in kwargs? That would keep the style consistent and provide feedback why the call will not work.

Reply:
Don't think you add any backwards compatibility over positional args, since if your code worked it supplied these arguments already.
```python
>>> def foo(**kwargs):
...     if 'one' not in kwargs:
...         raise Exception('missing param')
...     if 'two' not in kwargs:
...         raise Exception('missing param')
...     pass
... 
>>> def bar(one, two, **kwargs):
...     pass
... 
>>> 
>>> foo(one=1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 5, in foo
Exception: missing param
>>> bar(one=1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: bar() missing 1 required positional argument: 'two'
```

Reply:
I see. Positional arguments looks like the best path forward. Could you also modify the signature of get_sleep as well? That way we the sleep portion of the api is consistent.